### ISSUE TITLE
Typing error (page 472)

### ISSUE THREAD

**Issue by ashwinshetgaonkar** [rasbt/machine-learning-book#34]
print(f'CCE (w Probas): '              `#should be w Logits here`
 f'{cce_logits_loss_fn(logits, target):.4f}')
CCE (w Probas): 0.5996
print(f'CCE (w Logits): '              `#should be w Probas here`
 f'{cce_loss_fn(torch.log(probas), target):.4f}')
CCE (w Logits): 0.5996

**Comment by rasbt** [comment]
Good point, the "w Logits" and "w Probas" should be flipped. Thanks for the note, really appreciate it. Hope we can update the print copy to reflect that.

**Comment by rasbt** [comment]
Added it to the errata.
